.gitignore

```diff
@@ -18,6 +18,9 @@ _out/*
 
 # kubevirtCI
 _ci-configs/
+/cluster-up/
```

Comment:
It seems that you've committed the cluster-up directory though

Reply:
thanks for noticing that

Reply:
No, I have verified it, there is no cluster-up dir. 
What I have commited is a change that removes the cluter-up dir and all files there.